_config.yml

```diff
@@ -54,13 +55,13 @@ defaults:
       type: versions
     values: 
       layout: versions
-      permalink:  versions/:categories/:slug
+      permalink:  versions/:categories/:slug.html
```

Comment:
Good catch on these @AMoo-Miki.  Is there an SEO advantage to using raw paths (`/versions/foo/bar/`) vs files (`/versions/foo/bar.html`)? The site is pretty inconsistent when I step back and look at it.

Reply:
SEO-wise, none. The only reason I chose the `.html` was because the permalinks that visitors might have shared.

Reply:
Makes sense - given the name of the PR branch, I just wanted to confirm that we weren't doing anything counter to SEO best practices.